src/librustc_expand/proc_macro.rs

```diff
@@ -133,7 +133,7 @@ impl MultiItemModifier for ProcMacroDerive {
         };
 
         let error_count_before = ecx.parse_sess.span_diagnostic.err_count();
-        let msg = "proc-macro derive produced unparseable tokens";
+        let msg = "proc-macro derive produced unparsable tokens";
```

Comment:
This looks weird to me and Google search gives 3.5 times more hits for "unparseable" than for "unparsable".

Reply:
reverted as well (including corresponding tests)